ArchiSteamFarm/www/pages/command.html

```diff
@@ -210,11 +210,32 @@ <h3 class="box-title">Command</h3>
             var prevCommand = [];
             var commandCount = 0;
             var keyCount = 0;
+            var tmpPrevCommand = JSON.parse(get('prevCommand'));
+            var tmpCommandCount = get('commandCount');
+            var tmpKeyCount = get('keyCount');
+
+            if (tmpPrevCommand) {
+                prevCommand = tmpPrevCommand;
+            }
+
+            if (tmpCommandCount > 0) {
+                commandCount = tmpCommandCount
```

Comment:
forgot the semicolon here and on line 226 :)

Reply:
thanks, forgot about that!

how do you like the new command history? is it like you would have imagined? Is it working properly?

Reply:
Seems to work really nicely! 🥇 

Did you see https://github.com/MrBurrBurr/ArchiSteamFarm/pull/1 ?

btw you might want to set
`git config --global core.autocrlf true`